File: back/DevC/src/main/java/com/back/devc/domain/post/comment/attachment/dto/CommentAttachmentResponse.java
package com.back.devc.domain.post.comment.attachment.dto;

import java.time.LocalDateTime;

public record CommentAttachmentResponse(
        Long attachmentId,
        Long commentId,
        String fileName,
        String storedName,
        String fileUrl,
        String fileType,
        String mimeType,
        Long fileSize,
        Integer fileOrder,
        LocalDateTime createdAt
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/post/comment/attachment/dto/CommentAttachmentResponse.kt
package com.back.devc.domain.post.comment.attachment.dto

import java.time.LocalDateTime

data class CommentAttachmentResponse(
    val attachmentId: Long,
    val commentId: Long,
    val fileName: String,
    val storedName: String,
    val fileUrl: String,
    val fileType: String,
    val mimeType: String,
    val fileSize: Long,
    val fileOrder: Int,
    val createdAt: LocalDateTime,
)